src/core/MIGRATION.md

```diff
@@ -1240,7 +1240,7 @@ This table shows where these uiExports have moved to in the New Platform. In mos
 | `inspectorViews`             |                                                                                                                           | Should be an API on the data (?) plugin.                                                                                              |
 | `interpreter`                |                                                                                                                           | Should be an API on the interpreter plugin.                                                                                           |
 | `links`                      | n/a                                                                                                                       | Not necessary, just register your app via `core.application.register`                                                                 |
-| `managementSections`         | [`plugins.management.sections.register`](/rfcs/text/0006_management_section_service.md)                                   | API finalized, implementation in progress.                                                                                            |
+| `managementSections`         | [`plugins.management.sections.register`](/rfcs/text/0006_management_section_service.md)                                   |                                                                                                                                       |
```

Comment:
Could you also a row the table on line 1164 that shows where `ui/management` went?